src/main/webapp/queue.js

```diff
@@ -44,6 +65,44 @@ function updateShareTab() {
       'lounge.html/?room_id='+ roomParameters +'" type="text" readonly</input>';
 }
 
+/* exported voteToSkip */
+// Tracks the user's choice to skip a video and alters firestore as needed
+function voteToSkip() {
+  voteBtnCount++;
+  if (voteBtnCount % 2 == 1) {
+    db.collection('rooms') // eslint-disable-line no-undef
+        .doc(roomParameters)
+        .collection('CurrentVideo')
+        .doc('PlaybackData')
+        .update({
+          skipVotes: firebase // eslint-disable-line no-undef
+              .firestore.FieldValue.increment(1),
```

Comment:
would this cause race condition when two users vote at the exact time?

Reply:
It shouldn't because it will take whatever value is stored in firestore and increment it. Since I'm only dealing with integer values, this request is entirely dependent on client-server communication. Firestore only updates documents about once per second, so this shouldn't result in a race condition since requests will be queued. Let me know if you have any thoughts about this.